src/Symfony/Bundle/SecurityBundle/DependencyInjection/MainConfiguration.php

```diff
@@ -251,6 +251,18 @@ private function addFirewallsSection(ArrayNodeDefinition $rootNode, array $facto
                     ->scalarNode('path')->defaultValue('/logout')->end()
                     ->scalarNode('target')->defaultValue('/')->end()
                     ->booleanNode('invalidate_session')->defaultTrue()->end()
+                    ->scalarNode('clear_site_data')->defaultValue('off')->end()
+                    ->arrayNode('clear_site_data')
+                        ->fixXmlConfig('clear_site_data')
+                        ->performNoDeepMerging()
+                        ->defaultValue(['off'])
+                        ->beforeNormalization()->ifString()->then(function ($v) { return $v ? array_map('trim', explode(',', $v)) : []; })->end()
+                        ->enumPrototype()
+                        ->values([
```

Comment:
should be indented as it is about configuring the prototype node here (the rule is to indent for calls moving to a different builder object and to dedent on the `->end()` call moving back to the parent)